.github/workflows/reusable-plugin-prep-release.yml

```diff
@@ -219,54 +219,96 @@ jobs:
             "[.[] | select(.mergedAt > \"$TAG_DATE\") | \"- #\(.number) \(.title)\"] | join(\"\\n\")")
 
           if [ -z "$MERGED_PRS" ]; then
-            echo "No pull requests merged since last release."
+            echo "No pull requests merged since last release. Skipping PR listing."
             echo "prs=No pull requests merged since last release to the ${PLUGIN_SOURCE_BRANCH} branch." >> "$GITHUB_OUTPUT"
             exit 0
           fi
 
           {
-            echo "prs=<<EOF"
+            echo "prs<<EOF"
             echo "$MERGED_PRS"
             echo "EOF"
           } >> "$GITHUB_OUTPUT"
 
-      # create the release branch, commit the changes, and create pull request
-      - name: Create Release Pull Request
-        id: create_pr
-        uses: peter-evans/create-pull-request@271a8d0340265f705b14b6d32b9829c1cb33d45e  # v7.0.8
-        with:
-          token: ${{ github.token }}
-          base: ${{ inputs.plugin-target-branch }}
-          branch: release/${{ steps.new.outputs.version }}
-          title: "Automated ${{ inputs.level }} release: ${{ steps.new.outputs.version }}"
-          body: |
-            ## Changes in this pull request
+      # create the release branch, commit the changes with the github-actions[bot]
+      - name: Create and commit to release branch
+        id: push-changes
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OLD_VERSION: ${ steps.old.outputs.version }}
+          NEW_VERSION: ${{ steps.new.outputs.version }}
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
 
-            - Bumped version from `${{ steps.old.outputs.version }}` to `${{ steps.new.outputs.version }}`
-            - Updated build files
-            - Updated language files
+          # Use GitHub token to authenticate the push
+          git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}
 
-            ## Merged PRs since last release
+          RELEASE_BRANCH="release/${NEW_VERSION}"
+          git checkout -b "$RELEASE_BRANCH"
+          git add .
+          git commit -m "🤖 prep for release - ${OLD_VERSION} → ${NEW_VERSION}"
+          git push origin "$RELEASE_BRANCH"
 
-            ${{ steps.merged-prs.outputs.prs }}
+      # Create the PR. (Requires jq present on GitHub-hosted runners)
```

Comment:
Let's ditch this comment. `jq` is standard on the GitHub runners now and the step name mentions creating a PR.